LocalSettings.php

```diff
@@ -2381,23 +2379,16 @@
 			'managewiki-editdefault',
 			'mwoauthmanageconsumer',
 			'mwoauthmanagemygrants',
-			'mwoauthproposeconsumer',
 			'mwoauthsuppress',
-			'mwoauthupdateownconsumer',
 			'mwoauthviewprivate',
 			'mwoauthviewsuppressed',
-			'read',
 			'renameuser',
 			'requestwiki',
 			'siteadmin',
-			'skipcaptcha',
 			'suppressionlog',
 			'suppressrevision',
-			'torunblocked',
```

Comment:
allows people to whitelist tor for everyone.

Reply:
@JohnFLewis TorBlock was only made a global extension because of LP - prior to that it was an optional opt-in extension. I don’t think we should be forcing users to block Tor unless its a problem for them

Reply:
@JohnFLewis notice “Open proxies (or similar networks, such as web hosts) that MAY be used abusively“. Not all Tor networks will be used abusively.

Web hosts is one area where I feel that we should only be reactive and block when abused, not block every single one because of the chance of abuse

Reply:
Tor are open proxies, anyone can use them. Further anyone can abuse them and they are infamous in being even harder to predict usage and prevent abuse than even open proxies on the surface web.

You have an issue with a global policy, RfC is the place, not here. This change won't be merged until this right is readded or another valid non-policy based argument is presented.